stable/opsportal/values.yaml

```diff
@@ -50,6 +50,13 @@ opsportalRBAC:
   enabled: true
```

Comment:
nit: should we make rbac `false` by default,  for now? It is anyway enabled in `Kubernetes-base-addons`

Reply:
Let's leave the chart values at what is recommended. We can override that in Addons if we want to maintain compatibility with previous deployments.